SparkPlugs/main/default/aura/SparkPlugExampleForProcessRefund/SparkPlugExampleForProcessRefund.cmp-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<AuraDefinitionBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>59.0</apiVersion>
```

Comment:
Let's add description what is a minimum api version

Reply:
no requirements